src/Tools/ExternalAccess/FSharp/NavigateTo/FSharpNavigateToItemKind.cs

```diff
@@ -13,7 +13,6 @@ internal static class FSharpNavigateToItemKind
         public static string Line => NavigateToItemKind.Line;
         public static string File = NavigateToItemKind.File;
         public static string Class => NavigateToItemKind.Class;
-        public static string Record => NavigateToItemKind.Record;
```

Comment:
This is unfortunately external access, but it won't break FSharp since this isn't used on their end.

https://github.com/dotnet/fsharp/blob/d823b21a549b6c4dacf749f404e4db829844b3dd/vsintegration/src/FSharp.Editor/Navigation/NavigateToSearchService.fs#L118-L129